src/types/model/base/camunda_execution_listener.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Describes a single camunda execution listener.
```

Comment:
So here I am entirely not sure anymore but I would write it like this.

```suggestion
 * Describes a single Camunda specific ExecutionListener.
```